shuttle/templates/request_shuttle.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'shuttle_base.html' %}
+<!--TODO: Restrict roles to Bethel people-->
+<!--TODO: Have actual locations and times for Bethel shuttle. Also actual waitlist numbers-->
+{% block body_content %}
+
+    <div class="wrapper">
+        <h4>Select a location and time below to request the shuttle:</h4>
+    </div>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
```

Comment:
You should be able to add these scripts in the scripts block in `shuttle_base.html`. Also there is a bootstrap import already there but replace it with yours since yours is a newer version